resources/dicts/thoughts/alive/general.json

```diff
@@ -4,6 +4,7 @@
         "thoughts": [
             "Is feeling quite lazy",
             "Is spending a considerable amount of time grooming",
+            "Is spending a considerable amount of time grooming",
```

Comment:
a duplicate thought?

Reply:
I think I added it twice since cats do spend a lot of time grooming and thought it would make a bit of sense to appear slightly more often but yeah now that i think about it i think ill remove it :)